tests/test_material.py

```diff
@@ -1981,83 +1974,81 @@ def test_material_gammas():
-            (275.49, 4.0425668777137595e-20),
-            (281.42, 7.579812895713299e-21),
-            (282.92, 7.579812895713299e-21),
-            (289.56, 8.843115044998849e-21),
+            (182.62, 4.9268783822136435e-21),
+            (185.715, 7.200822250927634e-19),
+            (194.94, 7.958803540498964e-21),
+            (198.9, 4.547887737427978e-22),
+            (202.12, 1.3643663212283936e-20),
+            (205.316, 6.341776789413459e-20),
+            (215.28, 3.6635762329280945e-22),
+            (221.386, 1.4906965361569485e-21),
+            (228.78, 8.843115044998849e-23),
+            (233.5, 4.800548167285089e-22),
+            (240.88, 9.348435904713068e-22),
+            (246.83, 6.948161821070524e-22),
+            (266.45, 7.579812895713299e-23),
+            (275.35, 6.442840961356303e-22),
+            (275.49, 4.0425668777137595e-22),
+            (281.42, 7.579812895713299e-23),
+            (282.92, 7.579812895713299e-23),
+            (289.56, 8.843115044998849e-23),
             (291.2, np.nan),
-            (291.65, 5.053208597142199e-20),
-            (301.7, 6.3165107464277484e-21),
-            (317.1, 1.2633021492855498e-21),
-            (343.5, 3.7899064478566495e-21),
-            (345.92, 5.053208597142199e-20),
-            (356.03, 6.3165107464277484e-21),
-            (387.84, 5.053208597142199e-20),
-            (410.29, 3.7899064478566495e-21),
+            (291.65, 5.053208597142199e-22),
+            (301.7, 6.316510746427748e-23),
+            (317.1, 1.2633021492855498e-23),
+            (343.5, 3.7899064478566497e-23),
+            (345.92, 5.053208597142199e-22),
+            (356.03, 6.316510746427748e-23),
+            (387.84, 5.053208597142199e-22),
+            (410.29, 3.7899064478566497e-23),
             (428.7, np.nan),
-            (448.4, 1.2633021492855498e-21),
-            (49.55, 3.018820986575914e-19),
-            (113.5, 4.811245947355363e-20),
+            (448.4, 1.2633021492855498e-23),
+            (49.55, 3.018820986575914e-21),
+            (113.5, 4.811245947355363e-22),
         ],
     )
 
 
 def test_material_xrays():
     leu = {"U238": 0.96, "U235": 0.04}
     mat = Material(leu)
-    assert (
-        mat.xrays() ==
-        [
-            (93.35, 7.135646069465136e-18),
-            (89.953, 4.4112564001433475e-18),
-            (105.0, 3.394789326064896e-18),
-            (13.0, 2.9652250902501514e-17),
-            (93.35, 5.2767097753586244e-21),
-            (89.953, 3.2620619831267015e-21),
-            (105.0, 2.510398896994686e-21),
-            (13.0, 3.4433858446293e-17),
-        ])
+    assert mat.xrays() == [
+        (93.35, 7.135646069465136e-18),
+        (89.953, 4.4112564001433475e-18),
+        (105.0, 3.394789326064896e-18),
+        (13.0, 2.9652250902501514e-17),
+        (93.35, 5.2767097753586244e-21),
+        (89.953, 3.2620619831267015e-21),
+        (105.0, 2.510398896994686e-21),
+        (13.0, 3.4433858446293e-17),
+    ]
```

Comment:
Is the x-ray data also affected by the factor of 100?

Reply:
No. I think it occurred due to the use of black code formatter. The values are same.

Reply:
Understood, just wanted to make sure it wasn't an oversight :+1: